packages/components/src/Enumeration/Items/Item/Item.test.js

```diff
@@ -64,6 +64,24 @@ describe('Item', () => {
 		expect(props.item.itemProps.onSelectItem).toBeCalled();
 	});
 
+	it('should display value with only button which are not disabled', () => {
+		// given
+		const itemWithDisabled = { ...item };
+		itemWithDisabled.itemProps.actions[0].disabled = true;
```

Comment:
You change the action from the same reference as all the tests.
Please don't mutate it as it can impact all the following tests. --> deepClone.